hw/ip/aes/doc/checklist.md

```diff
@@ -75,8 +75,8 @@ Security      | [SEC_RND_CNST][]        | Done        |
 
  Type         | Item                         | Resolution  | Note/Collaterals
 --------------|------------------------------|-------------|------------------
-Security      | [SEC_CM_IMPLEMENTED][]       | Not Started |
-Security      | [SEC_CM_ASSETS_LISTED][]     | Not Started |
+Security      | [SEC_CM_IMPLEMENTED][]       | Done        | Datapath FI hardening in progress (not strictly required)
```

Comment:
There is still some review backlog for other AES changes. At the same, datapath FI hardening has never been a strict requirement for the first version. We've thus decided to take AES to D2S before datapath FI hardening is complete and continue the implementation in parallel.